sensor_msgs/RegionOfInterest[] roi
int8[] clsID
std_msgs/String[] clsName
float32[] confidence
Metrics[] metrics